### DISCUSSION TITLE
Quiz: 1.2 How Can We Use Machine Learning? Part 1

### DISCUSSION THREAD

**Discussion by PatTheAtak** [Lightning-AI/dl-fundamentals#26] [Exercises] [answered]
`Using machine learning, we can compress a 784-dimensional handwritten image into …`

When compressing a 784 -dimensional handwritten image into a 1 - dimensional image, wouldn't the compressed representation lose too much information making it lose it's usefulness, and it would not be possible to reconstruct the original image from the compressed version.

**Comment by rasbt** [comment, marked as answer]
Sorry, I missed this somehow! You are right, it would be a lossy compression (like PNG -> JPEG)... but a compression nonetheless :P.
